examples/resources/security_policy/resource_cluster_custom_security_policy.tf

```diff
@@ -0,0 +1,136 @@
+# Create Tanzu Mission Control security policy
```

Comment:
can you add more description in these comments. Like two lines not to much. 

same changes in other places

Reply:
Sure

Reply:
Done